tests/test_validate.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf8 -*-
+"""Tests for the metsrw.validate module."""
+
+from __future__ import print_function
+
+import os
+
+import pytest
+
+import metsrw
+
+
+def test_get_schematron(mocker):
+    bad_path = 'some/nonexistent/path/'
+    with mocker.patch('metsrw.get_file_path') as get_file_path:
+        with pytest.raises(ValueError):
+            metsrw.get_schematron(bad_path)
+            get_file_path.assert_called_once_with(bad_path)
+
+    with mocker.patch('metsrw.get_file_path') as get_file_path:
+        mocker.patch.object(os.path, 'isfile', return_value=True)
+        with pytest.raises(IOError):
+            metsrw.get_schematron(bad_path)
+            get_file_path.assert_called_once_with(bad_path)
+
+    def mockisfile(path):
+        if path == bad_path:
+            return False
+        return True
```

Comment:
Just curious, is this the Pythonic way of `return not path == bad_path`? I like explicit, just wondering!

Reply:
I think what you've written is equally Pythonic. Maybe I'll add some parens so nobody has to think about operator precedence: `return not (path == bad_path)`.